virtual-organizations/OSG.yaml

```diff
@@ -181,6 +181,17 @@ DataFederations:
         AllowedCaches:
           - ANY
 
+      - Path: /nrdstor/protected
+        Authorizations:
+          - SciTokens:
+              Issuer: https://t.nationalresearchplatform.org
```

Comment:
What issuer is this?

Reply:
@biozit can you drop a comment here explaining this issuer for posterity?